src/Components/Blazor/BlazorExtension/src/Microsoft.VisualStudio.BlazorExtension.csproj

```
@@ -26,6 +26,9 @@
     <GeneratePkgDefFile>true</GeneratePkgDefFile>
     <DeployProjectOutput Condition="'$(ContinuousIntegrationBuild)' == 'true'">false</DeployProjectOutput>
     <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
+
+    <!-- The Resources.resx is a embedded resource -->
+    <GenerateResxSource>true</GenerateResxSource>
```

The comment seems to indicate you intended this to be
```suggestion
    <GenerateResxSource>false</GenerateResxSource>
```